### PR TITLE
[Notifier] [Firebase] Only call setMessageId if valid message id

### PR DESCRIPTION
**Pull request by OskarStark**
| Q             | A
| ------------- | ---
| Branch?       | 5.3
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | Follows https://github.com/symfony/symfony/pull/43073#issuecomment-922526653
| License       | MIT
| Doc PR        | ---

### CODE REVIEW

File: src/Symfony/Component/Notifier/Bridge/Firebase/FirebaseTransport.php [hunk 1]
```diff
@@ -96,7 +96,10 @@ protected function doSend(MessageInterface $message): SentMessage
         $success = $response->toArray(false);
 
         $sentMessage = new SentMessage($message, (string) $this);
-        $sentMessage->setMessageId($success['results'][0]['message_id'] ?? '');
+
+        if (null !== $success['results'][0]['message_id'] ?? null) {
```
Comment (noniagriconomie) [inline src/Symfony/Component/Notifier/Bridge/Firebase/FirebaseTransport.php:100]:
```suggestion
        if (null !== $messageId = $success['results'][0]['message_id'] ?? null) {
            $sentMessage->setMessageId($messageId);
```

Comment (OskarStark) [reply src/Symfony/Component/Notifier/Bridge/Firebase/FirebaseTransport.php:100]: I am not a big fan of assigning inside a comparison, plus this variable is not used anymore, so one time variables can be avoided

Comment (noniagriconomie) [reply src/Symfony/Component/Notifier/Bridge/Firebase/FirebaseTransport.php:100]: yeah indeed, matter of taste :)

**Comment by fabpot** [comment]
As I expect that most the time we will have a message id, I don't think this is needed.

**Pull request closed without merging**